orchestra/bots/staffbot.py

```diff
@@ -142,7 +199,11 @@ def send_staffing_request_by_slack(self, staffing_request, message):
                 worker))
             return
 
-        self.slack.chat.post_message(
-            worker.slack_user_id, message, parse='none')
+        if settings.PRODUCTION:
```

Comment:
let's move this into the slack object so we don't have to repeat the logic elsewhere